person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def can_use_services?
+    is_of_age? || @parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] Kindly check your `of_age?` method and where you are calling it on line `13`. Your method name is `of_age?` but you are calling it `is_of_age?` on line 13. Kindly fix this by naming the method `is_of_age?` and not `of_age?`.

Reply:
I rename it because it gives a linter error I will try to rename it again

Reply:
![image](https://user-images.githubusercontent.com/89970442/158486072-cba4d856-d263-49ab-b555-c5d35c0291c4.png)